resources/dd-ix.net.zone

```diff
@@ -0,0 +1,60 @@
+; Check me with (abs. path required....)
+; named-checkzone dd-ix.net /.../resources/dd-ix.net.zone
+
+$TTL    1h         ; default TTL for zone
+$ORIGIN dd-ix.net. ; base domain-name
+
+@  3600  IN  SOA  ns.dd-ix.net  noc.dd-ix.net.  (
+        2020080302  ;Serial
```

Comment:
Do we keep the usual serial schema (`YYYYMMDDnn`)? Please bump ;)

Reply:
Just Copy Pasted and forgot to change